packages/phrases/src/locales/de/errors/oidc.ts

```diff
@@ -1,29 +1,24 @@
 const oidc = {
   aborted: 'Der Endnutzer hat die Interaktion abgebrochen.',
-  /** UNTRANSLATED */
-  invalid_scope: 'Invalid scope: {{error_description}}.',
+  invalid_scope: 'Ungültiger Anwendungsbereich: {{error_description}}.',
   invalid_token: 'Ungültiger Token übermittelt.',
   invalid_client_metadata: 'Ungültige Client Metadaten übermittelt.',
-  /** UNTRANSLATED */
-  insufficient_scope: 'Token missing scope `{{scope}}`.',
+  insufficient_scope: 'Token fehlt den Anwendungsbereich `{{scope}}`.',
   invalid_request: 'Anfrage ist ungültig.',
-  invalid_grant: 'Grant request ist ungültig.',
+  invalid_grant: 'Grant Anfrage ist ungültig.',
   invalid_redirect_uri:
     '`redirect_uri` stimmt nicht mit den registrierten `redirect_uris` des Clients überein.',
   access_denied: 'Zugang verweigert.',
-  invalid_target: 'Ungültiger resource indicator.',
+  invalid_target: 'Ungültiger Ressourcenindikator.',
```

Comment:
Should we worry about these capitalization issues?

Reply:
actually i have no idea about if it's a language rule from Germany or we can use lowercase. i prefer to keep as it is